firmware/quadruna/CRIT/src/app/app_globals.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "io_switch.h"
+#include "io_rgbLed.h"
+#include "io_led.h"
+
+typedef struct
+{
+    const BinaryLed *const imd_led;
+    const BinaryLed *const bspd_led;
+    const BinaryLed *const ams_led;
+    const BinaryLed *const start_led;
+    const BinaryLed *const regen_led;
+    const BinaryLed *const torquevec_led;
+    const BinaryLed *const shdn_led;
+    const Switch *const    start_switch;
+    const Switch *const    regen_switch;
+    const Switch *const    torquevec_switch;
+    const RgbLed *const    bms_status_led;
+    const RgbLed *const    fsm_status_led;
+    const RgbLed *const    vc_status_led;
+    const RgbLed *const    aux_status_led;
+    const RgbLed *const    crit_status_led;
+    const RgbLed *const    rsm_status_led;
+} GlobalsConfig;
+
+typedef struct
+{
+    const GlobalsConfig *config;
+} Globals;
+
+extern Globals *const globals;
```

Comment:
you might have to fact check me on this, but i think we can `extern const crit_globals` (which can be renamed to just `globals`) while making `crit_globals` non-constant in the source file (.c file)

Reply:
Should be a pointer.